core/src/metrics.rs

```diff
@@ -63,6 +84,14 @@ pub(crate) fn register_port_stats(ports: &[Port]) {
     });
 }
 
+/// Registers collected mempool stats with the metrics store.
+pub(crate) fn register_mempool_stats<'a>(mempools: impl Iterator<Item = &'a Mempool>) {
+    let stats = mempools.map(Mempool::stats).collect::<Vec<_>>();
+    SINK.clone().proxy("mempool", move || {
```

Comment:
reading the docs, SINK's can be cheaply cloned yeah? I should have brought this up in a previous PR.

Reply:
yeah. relatively cheap as long as we are not doing it in a critical path like per packet. this should be fine.